rom/dev/src/flow/cold_reset/fw_processor.rs

```diff
@@ -570,6 +588,43 @@ impl FirmwareProcessor {
         (nb, nf)
     }
 
+    /// Read request from mailbox and verify the checksum
+    ///
+    /// # Arguments
+    /// * `txn` - Mailbox Receive Transaction
+    /// * `data` - Data buffer for the expected request
+    ///
+    /// # Returns
+    /// * `()` - Ok
+    ///    Error code on failure.
+    fn copy_req_verify_chksum(txn: &mut MailboxRecvTxn, data: &mut [u8]) -> CaliptraResult<()> {
+        // TODO: Some commands may have a variable size with a max instead of a fixed size
```

Comment:
Is there an additional change needed to address this TODO?

Reply:
Good question. None of the commands we support in ROM currently have a variable request size, so it's not needed at the moment. I can change the wording on this so it's clearer this is sufficient for the current command set.